Mathlib/Topology/MetricSpace/Lipschitz.lean

```diff
@@ -118,6 +122,11 @@ theorem MapsTo.lipschitzOnWith_iff_restrict [PseudoEMetricSpace α] [PseudoEMetr
 alias ⟨LipschitzOnWith.to_restrict_mapsTo, _⟩ := MapsTo.lipschitzOnWith_iff_restrict
 #align lipschitz_on_with.to_restrict_maps_to LipschitzOnWith.to_restrict_mapsTo
 
+/-- `f : α → β` is called **locally Lipschitz continuous** iff every point `p ∈ α`
```

Comment:
```suggestion
/-- `f : α → β` is called **locally Lipschitz continuous** iff every point `x`
```
since there is no `∈` in the definition, and there you used `x` not `p` (which matches most of the file)